sdk/rust/oak/src/grpc/mod.rs

```diff
@@ -117,14 +117,14 @@ impl ChannelResponseWriter {
 
     /// Close out the gRPC method invocation with the given final result.
     pub fn close(&self, result: Result<()>) -> std::result::Result<(), OakError> {
-        // Build a final GrpcResponse message wrapper and serialize it into the
-        // channel.
-        let mut grpc_rsp = GrpcResponse::default();
-        grpc_rsp.last = true;
         if let Err(status) = result {
+            // Build a final GrpcResponse message wrapper and serialize it into the
```

Comment:
To me these changes seem reasonable, please let me know if this causes problems somewhere else (e.g. if anyone relies on `last` always being set on the final message).

Reply:
This may be a separate change, could you split it into a separate PR, so it's not blocked on the rest of the review?